doc/whats-new.rst

```diff
@@ -14,10 +14,24 @@ What's New
 
     np.random.seed(123456)
 
-.. _whats-new.2023.08.1:
+.. _whats-new.2023.09.0:
 
-v2023.08.1 (unreleased)
------------------------
+v2023.09.0 (Sep XX, 2023)
+-------------------------
+
+This release continues work on the new :py:class:`xarray.Coordinates` object, allows to provide `preferred_chunks` when
+reading from netcdf files, enables :py:func:`xarray.apply_ufunc` to handle missing core dimensions and fixes several bugs.
+
+Thanks to the 24 contributors to this release: Alexander Fischer, Amrest Chinkamol, Benoit Bovy, Darsh Ranjan, Deepak Cherian,
+Gianfranco Costamagna, Gregorio L. Trevisan, Illviljan, Joe Hamman, JR, Justus Magin, Kai Mühlbauer, Kian-Meng Ang, Kyle Sunden,
+Martin Raspaud, Mathias Hauser, Mattia Almansi, Maximilian Roos, András Gunyhó, Michael Niklas, Richard Kleijn, Riulinchen,
+Tom Nicholas, Wiktor Kraśnicki.
+
+We welcome the following new contributors to Xarray!: Alexander Fischer, Darsh Ranjan, Gianfranco Costamagna, Gregorio L. Trevisan,
+Kian-Meng Ang and Wiktor Kraśnicki.
```

Comment:
Should the authors of #7578 and #7955 be included here?

Reply:
Yes, sure. Thanks for catching this, @keewis.

Reply:
I've added #7955 to the docs section, it had no entry in whats-new.rst.